.github/scripts/dapr_bot.js

```diff
@@ -157,7 +158,7 @@ async function handleIssueLabeled({ github, context }) {
     // Only authorized users can add labels to issues.
     if (label == 'docs-needed') {
         // Open a new issue
-        await github.issues.create({
+        await github.rest.issues.create({
```

Comment:
What is the significance of `github.issues` -> `github.rest.issues`, or is this the new path in v6?

Reply:
That’s how v5+ work. They changed the APIs.